src/assets/templates/widgets/enrollment/notifications.html

```diff
@@ -18,7 +18,7 @@
       </li>
       <li>
         LLM and JSD students should contact the Advanced Degree Program Office at
-        <a href="mailto:llmadmits@law.berkeley.edu">llmadmits@law.berkeley.edu</a>.
+        <a href="mailto: adpoffice@law.berkeley.edu">adpoffice@law.berkeley.edu</a>.
```

Comment:
@nkutub there is a space between the `:` and `adpoffice`.  Is that intentional?

Reply:
oops , I will fix